src/test/java/pt/compta/tools/fourthmaven/MathUtilsTest.java

```diff
@@ -9,10 +9,18 @@
 public class MathUtilsTest {
 
```

Comment:
So, I notice that the old method is missing. The intention here was to Add more tests not Replace. Could you add back the missing tests? Thanks

Reply:
Just inserted the original content as it was requested.